libs/common/src/helpers/schemas/z-boolean-coerce.test.ts

```diff
@@ -0,0 +1,59 @@
+import { zBooleanCoerce } from './z-boolean-coerce';
+
+describe('zBooleanCoerce', () => {
+  describe('when parsing a value with zBooleanCoerce', () => {
+    describe('when the provided value can be coerced to a boolean', () => {
+      it.each(getSuccessTestCases())('should return $result when the provided value is $description', ({ value, result }) => {
+        expect(zBooleanCoerce.parse(value)).toBe(result);
+      });
+    });
+
+    describe('when the provided value cannot be coerced to a boolean', () => {
+      it.each(getFailureTestCases())('should throw an error when the provided value is $description', ({ value }) => {
+        expect(() => zBooleanCoerce.parse(value)).toThrow();
```

Comment:
fwiw could ensure we're not getting an error for another reason
```suggestion
        expect(() => zBooleanCoerce.parse(value)).toThrow(ZodError);
```